tests/tests.go

```diff
@@ -728,6 +728,33 @@ func (ctl *controller) patchTaskStatus(taskStatusPatch api.TaskStatusPatch, pipe
 	return task, nil
 }
 
+// patchTaskStatus patches the status of all tasks in the pipeline stage.
+func (ctl *controller) patchStageTasksStatus(stageTasksStatusPatch api.StageTasksStatusPatch, pipelineID int) ([]*api.Task, error) {
+	buf := new(bytes.Buffer)
+	if err := jsonapi.MarshalPayload(buf, &stageTasksStatusPatch); err != nil {
+		return nil, fmt.Errorf("failed to marshal stageTasksStatusPatch, error: %w", err)
+	}
+
+	body, err := ctl.patch(fmt.Sprintf("/pipeline/%d/stage/%d/status", pipelineID, stageTasksStatusPatch.ID), buf)
+	if err != nil {
+		return nil, err
+	}
+
+	var tasks []*api.Task
+	ts, err := jsonapi.UnmarshalManyPayload(body, reflect.TypeOf(new(api.Task)))
```

Comment:
tasks

ts is ambiguous

Reply:
Done.

Used untypedTasks instead to avoid conflicting with other tasks which are typed.